src/tests/ngraph_helpers/ngraph_functions/include/ngraph_functions/utils/data_utils.hpp

```diff
@@ -35,6 +35,12 @@ generateVector(size_t vec_len,
             res[i] = static_cast<dataType>(dist(gen));
         }
         return res;
+    } else if (std::is_same<bool, dataType>()) {
+        std::bernoulli_distribution dist;
```

Comment:
Minor: comment why we use such code here will be usable

Reply:
You suggesting to copy-paste a definition of `std::bernoulli_distribution` here?